estate/models/estate_property.py

```diff
@@ -40,4 +40,14 @@ def _default_date(self):
         copy = False, 
         default = "new",
     )
-    
\ No newline at end of file
+    property_type_id = fields.Many2one("estate.property.type", string = "Property Type")
+    buyer_id = fields.Many2one("res.partner", string = "Buyer", copy = False)
+    salesperson_id = fields.Many2one('res.users', string = "Salesperson", default = lambda self: self.env.user)
+    offer_ids = fields.One2many("estate.property.offer", "property_id", string = "Offers")
+    tag_ids = fields.Many2many(
+        'estate.property.tag',  # The related model (assuming this is your tag model)
+        'estate_property_tag_rel',  # Explicitly define the name of the relation table
+        'property_id',  # The column in the relation table referring to this model
+        'tag_id',  # The column in the relation table referring to the tag model
+        string='Tags'
+    )
```

Comment:
Comme discuté au call ce matin
```suggestion
    tag_ids = fields.Many2many(
        'estate.property.tag',  # The related model (assuming this is your tag model)
        string='Tags'
    )
```

Reply:
> Comme discuté au call ce matin

@kfr-odoo je travaillé déjà sur une autre branch, ces modifications vont venire dans le chapitre 8, bientôt